app/assets/splitpane.css

```diff
@@ -11,8 +11,10 @@
 }
 
 .Resizer:hover {
-  -webkit-transition: all 2s ease;
-  transition: all 2s ease;
+  -webkit-transition: margin 0s;
+  transition: margin 0s;
```

Comment:
CSS renderer will take only the last declaration into account,  so I will remove the duplicates.